bootstrap/http/src/test/kotlin/com/studentcenter/weave/bootstrap/integration/MeetingAttendanceCreateApplicationServiceIntegrationTest.kt

```diff
@@ -0,0 +1,250 @@
+import com.studentcenter.weave.application.meetingTeam.service.domain.MeetingTeamDomainService
+import com.studentcenter.weave.application.user.vo.UserAuthenticationFixtureFactory
+import com.studentcenter.weave.domain.meeting.entity.MeetingAttendance
+import com.studentcenter.weave.domain.meeting.entity.MeetingFixtureFactory
+import com.studentcenter.weave.domain.meeting.enums.MeetingStatus
+import com.studentcenter.weave.domain.meetingTeam.entity.MeetingMember
+import com.studentcenter.weave.domain.meetingTeam.entity.MeetingTeam
+import com.studentcenter.weave.domain.meetingTeam.entity.MeetingTeamFixtureFactory
+import com.studentcenter.weave.domain.meetingTeam.enums.MeetingMemberRole
+import com.studentcenter.weave.domain.meetingTeam.enums.MeetingTeamStatus
+import com.studentcenter.weave.domain.user.entity.UserFixtureFactory
+import com.studentcenter.weave.domain.user.enums.Gender
+import com.studentcenter.weave.support.common.exception.CustomException
+import com.studentcenter.weave.support.common.uuid.UuidCreator
+import com.studentcenter.weave.support.security.context.SecurityContextHolder
+import io.kotest.assertions.throwables.shouldThrow
+import io.kotest.core.annotation.DisplayName
+import io.kotest.matchers.shouldBe
+
+@DisplayName("MeetingAttendanceCreateApplicationService Integration Test")
+class MeetingAttendanceCreateApplicationServiceIntegrationTest(
+    private val meetingDomainService: MeetingDomainService,
+    private val meetingAttendanceDomainService: MeetingAttendanceDomainService,
+    private val meetingTeamDomainService: MeetingTeamDomainService,
+    private val sut : MeetingAttendanceCreateApplicationService,
+) : IntegrationTestDescribeSpec({
+    val user = UserFixtureFactory.create()
+    val userAuthentication = UserAuthenticationFixtureFactory.create(user)
+    var userTeam: MeetingTeam? = null
+    var userTeamMember: MeetingMember? = null
+
+    fun createMeetingTeamAndMembers(
+        memberCount: Int,
+        gender: Gender,
+        isUserTeam: Boolean = false,
+    ): Pair<MeetingTeam, List<MeetingMember>> {
+        return MeetingTeamFixtureFactory.create(
+            status = MeetingTeamStatus.PUBLISHED,
+            memberCount = memberCount,
+            gender = gender
+        ).let { team ->
+            meetingTeamDomainService.save(team)
+            if (isUserTeam) userTeam = team
+            val members = IntRange(0, memberCount - 1).map { i ->
+                val memberUser = if (isUserTeam && i == 0) user
+                else UserFixtureFactory.create(gender = gender)
+                val role = if (i == 0) MeetingMemberRole.LEADER else MeetingMemberRole.MEMBER
+                val member = meetingTeamDomainService.addMember(memberUser, team, role)
+                if (isUserTeam && i == 0) userTeamMember = member
+                member
+            }.toList()
+
+            team to members
+        }
+    }
```

Comment:
p4)

```suggestion
    fun createMeetingTeamAndMembers(
        memberCount: Int,
        gender: Gender,
        isUserTeam: Boolean = false
    ): Pair<MeetingTeam, List<MeetingMember>> {
        val team: MeetingTeam = MeetingTeamFixtureFactory.create(
            status = MeetingTeamStatus.PUBLISHED,
            memberCount = memberCount,
            gender = gender
        ).also { team ->
            meetingTeamDomainService.save(team)
            if (isUserTeam) userTeam = team
        }

        val members: List<MeetingMember> = List(memberCount) { index ->
            val isLeader: Boolean = index == 0
            val memberUser: User =
                if (isUserTeam && isLeader) user else UserFixtureFactory.create(gender = gender)
            val role: MeetingMemberRole =
                if (isLeader) MeetingMemberRole.LEADER else MeetingMemberRole.MEMBER

            meetingTeamDomainService
                .addMember(memberUser, team, role)
                .also { member -> if (isUserTeam && isLeader) userTeamMember = member }
        }

        return team to members
    }
```

이렇게 member 부분은 분리하는게 더 깔끔할것 같은데 어떤가요?

Reply:
일단 로직 자체는 훨씬 깔끔해진 것 같아서 좋네요! 👍

Reply:
Fixture 사용은 좀 고민이에요! 어떤 식으로 정리해야될지?!
실제 객체만 생성하는 것객체 생성 및 영속화까지 해야하는 것 혹은 그 외에 사이드 이펙트까지 처리되도록 하기

Reply:
fixture에서 객체 생성만 하고, 영속화는 테스트에서 하는게 맞는것 같아요 - 중복 나오면 그때 fixture로 옮겨도 될듯?